CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 1.6.2
```

Comment:
Probably we do not want to release just with that as users are oblivious to this change, maybe just add "master" or something like that and not change the version either